lib/lacerda/compare/json_schema.rb

```diff
@@ -233,7 +240,7 @@ def _e(error, location, extra = nil)
       #
       def data_for_pointer(data_or_pointer, schema)
         data = nil
-        if data_or_pointer['type']
+        if data_or_pointer['type'] || data_or_pointer['oneOf']

```

Comment:
Before we ignored the `oneOf`s. This caused errors with `'oneOf' => [ { "$ref" => "#/definitions/post"] }` (added a test [here](https://github.com/moviepilot/lacerda/pull/43/files#diff-52c44d514913a06380abba99fcf86c7cR433)). This is the way usually lacerda transforms references.